stp_zmq/test/test_no_size_limit_for_outbound_msgs.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from stp_core.loop.eventually import eventually
+from stp_core.network.port_dispenser import genHa
+from stp_core.test.helper import CounterMsgsHandler, prepStacks
+from stp_zmq.test.helper import genKeys
+from stp_zmq.zstack import ZStack
+
+
+MSG_LEN_LIMIT = 250
+
+
+@pytest.fixture(params=[-1, 0, 1, 1000])
+def modifier(request):
+    return request.param
+
+
+def test_no_size_limit_for_outbound_msgs(looper, tdir, modifier):
+    names = ['Alpha', 'Beta']
+    genKeys(tdir, names)
+
+    alpha = ZStack(names[0], ha=genHa(), basedirpath=tdir, msgHandler=None, restricted=False)
+    alpha.msgLenVal.max_allowed = MSG_LEN_LIMIT
+
+    beta_msg_handler = CounterMsgsHandler()
+    beta = ZStack(names[1], ha=genHa(), basedirpath=tdir, msgHandler=beta_msg_handler.handler, restricted=False)
+
+    prepStacks(looper, alpha, beta)
+
+    def check_received(value):
+        assert beta_msg_handler.receivedMsgCount == value
+
+    msg_value_len = MSG_LEN_LIMIT - len("{'k':''}")
```

Comment:
I think, that we need to synchronize template `{'k':''}` and template which makes `msg`, `{'k': 'v'}`. I mean count of whitespaces and value. As i understand, this test needs for showing that we allow messages with size more then MSG_LEN_LIMIT. Maybe we should simplify this case for checking message with serialized length more and less then MSG_LEN_LIMIT?
I mean, that we can use parameterized fixture which return whole message with serialized length less and more then MSG_LEN_LIMIT.

Reply:
made some changes, hopefully it makes more sense now...